src/csvToJson.js

```diff
@@ -45,13 +46,30 @@ const assign = (file, dataEntries) => {
   }
 };
 
-// @TODO
-// I don't like the name for this method and for the whole file
-// if it's main - then let's put it into index.js
-const csvToJson = async (path, headers) => {
-  const fullPath = makeFullPath(path, '.csv');
-  const data = await parseCsv(fullPath, headers);
-  assign(path, data);
+/**
+ * @async
+ * @param {dirPath} dirPath directory path
+ * @param {data} data
+ * @returns {Promise<void>} Promise
+ */
+const csvToJson = (dirPath, data) => {
+  // stringify data with indent
+  const json = JSON.stringify(data, null, 2);
+
+  fs.readdir(dirPath, async (err, files) => {
```

Comment:
Can we replace this method with `read` method from generator?

Reply:
which method `fs.readdir` or `JSON.stringify`